test/e2e/autoscaling_utils.go

```diff
@@ -50,16 +54,18 @@ type ConsumingRC struct {
 	framework *Framework
 	channel   chan int
 	stop      chan int
+	expClient *client.ExperimentalClient
 }
 
 // NewConsumingRC creates new ConsumingRC
-func NewConsumingRC(name string, replicas int, framework *Framework) *ConsumingRC {
-	startService(framework.Client, framework.Namespace.Name, name, replicas)
+func NewConsumingRC(name string, replicas int, cpu string, framework *Framework) *ConsumingRC {
```

Comment:
I think we shouldn't mix ConsumingRC with HorizontalPodAutoscaler. Method createHorizontalPodAutoscaler should be moved to horizontal_pod_autoscaling.go and called directly from test. Argument cpu in this method should define initial cpu consumption of the rc.